app/domain/mail_relay/lists.rb

```diff
@@ -179,5 +181,33 @@ def strip_spam_headers(message)
       end
     end
 
+    def append_unsubscribe
+      if defined? mailing_list
+        if mailing_list.subscribable?
+          html?(message) ? append_unsubscribe_link : append_unsubscribe_text
+        end
+      end
+    end
+
+    def bulk_mail
+      append_unsubscribe
+      super
+    end
+
+    def html?(message)
+      message.multipart?
+    end
+
+    def append_unsubscribe_text
+      message.body = unsubscribe_url(false)
+    end
+
+    def append_unsubscribe_link
+      message.html_part = unsubscribe_url(true)
+    end
+
+    def unsubscribe_url(is_html)
```

Comment:
it's not pretty common to us the keyword 'is' in ruby.
`unsubscribe_url(html: false)`
use named argument here